modules/security-network-namespaces.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-network.adoc
+
+[id="security-network-namespaces_{context}"]
+= Using network namespaces
+
+{product-title} uses software-defined networking (SDN) to provide a unified
+cluster network that enables communication between containers across the
+cluster.
+
+Network policy mode, by default, makes all Pods in a project accessible from
+other Pods and network endpoints.
+Using multitenant mode, you can provide project-level isolation for pods and services.
+Pods become isolated by having a NetworkPolicy select them.
```

Comment:
"Pods become isolated by having a NetworkPolicy select them. "  It is also for network policy mode.  But just remove this sentence,  instead, using this sentence :   "To isolate one or more Pods in a project, you can create NetworkPolicy objects in that project to indicate the allowed incoming connections."    

So this paragraph like:
 Network policy mode, by default, makes all Pods in a project accessible from other Pods and network endpoints. To isolate one or more Pods in a project, you can create NetworkPolicy objects in that project to indicate the allowed incoming connections. 
Using multitenant mode, you can provide project-level isolation for pods and services.

Reply:
Done.